doc/code/qml_fermi.rst

```diff
@@ -51,18 +51,18 @@ The fermionic operators can be mapped to the qubit basis by using the
 Fermi sentences.
 
 >>> qml.jordan_wigner(qml.FermiA(1))
-(0.5*(PauliZ(wires=[0]) @ PauliX(wires=[1])))
-+ (0.5j*(PauliZ(wires=[0]) @ PauliY(wires=[1])))
+(0.5*(Z(0) @ X(1)))
++ (0.5j*(Z(0) @ Y(1)))
```

Comment:
With other updates to the repr, is this what it'll now look like?

Reply:
Ah yes good point, will have to update those too